runtime/vm/FastJNI_com_ibm_crypto_plus_provider_icc_NativeInterface.cpp

```diff
@@ -0,0 +1,415 @@
+/*******************************************************************************
+ * Copyright (c) 2001, 2020 IBM Corp. and others
```

Comment:
Should be 2020, 2020.

Reply:
Nah ... I just completed the required Developer Legal Education.  It should be 2001, 2020.   The first number is the year in which the first time commit or publish.  The second number is the year in which the latest modification is.

Reply:
is this completely new code?  then, it should be 2020,2020

Reply:
if this is a modification to some J9 code, it could be 2001,2020.

Reply:
This is a new file.